src/haxeLanguageServer/helper/ImportHelper.hx

```diff
@@ -66,17 +66,23 @@ class ImportHelper {
 		if (tokens == null) {
 			return null;
 		}
+
 		var firstImport = null;
-		var firstType = null;
+		var packageStatement = null;
+
+		// if the first token in the file is a comment, we should add the import after this
```

Comment:
I don't quite remember, is this the same as the previous behavior, or did that only skip the first comment if it's multiline (a license header)?

Reply:
I think it's just multi-line that it used to skip

We've still got the problem that it will do this:

```haxe
/**
    Doc
**/

import Test;

class Main {}
```

```haxe
/**
    Doc
**/
class Main {}
```

Because it can't differentiate between a license header and a class doc. I'm tempted to add a little herutric to check if there's a new line after the doc so it makes a better guess.

Will try that tomorrow

Reply:
Hm.. well, haxe-formatter removes the empty line in that case anyway. :)

Reply:
Ahh ok, I'll leave matching the original behavior then